presto-cli/src/main/resources/META-INF/native-image/jni-config.json

```diff
@@ -0,0 +1,173 @@
+[
```

Comment:
How did you generate this? Can you please add step to github actions that tries to generate them and see if there are no changes comparing to what was commited? To make sure this file is up to date. Something like: https://github.com/prestosql/presto/blob/master/presto-main/bin/check_webui.sh

Reply:
If we can generate them, we should do that at build time and not check it in at all.